client/components/global-notices/index.jsx

```diff
@@ -61,14 +62,15 @@ const NoticesList = React.createClass( {
 					onDismissClick={ this.removeNotice.bind( this, notice ) }
 					showDismiss={ notice.showDismiss }
 				>
-					{ notice.button &&
```

Comment:
FWIW I think the indentation was correct here previously, tabbed one on the children of the `Notice` even if that child is a condition.